server/text_generation_server/layers/tensor_parallel.py

```diff
@@ -144,7 +144,7 @@ def load_qkv(
             num_key_value_heads=num_key_value_heads,
         )
         if bias:
-            raise NotImplementedError("packed_qkv only implemented for baichuan")
+            bias = weights.get_tensor(f"{prefix}.bias")
```

Comment:
This is wrong, no ?

We get the whole bias on Row Parallel, for column, you need to take the actual slice, which for qkv you need to follow the same layout as the weights I think (except on dim=0 instead of dim=1)

Reply:
after comparing with transformers it seems like `weights.get_tensor(f"{prefix}.bias")` and `weights.get_sharded(f"{prefix}.qkv.bias", dim=0)` return the exact bias as the one in the reference.

I've reverted the change within `tensor_parallel.py::load_qkv` in favor of setting the bias after creating the linear in `qwen_vl.py` via `weights.get_sharded(f"{prefix}.qkv.bias", dim=0)`

for reference:
```python
self.qkv = TensorParallelColumnLinear.load_qkv(
    config,
    prefix=f"{prefix}.qkv",
    weights=weights,
    bias=False,
    num_heads=self.num_heads,
    num_key_value_heads=self.num_heads,
)
self.qkv.linear.bias = weights.get_sharded(f"{prefix}.qkv.bias", dim=0)
```

this hopefully makes the qkv loading a bit more clear